kyaml/setters2/add.go

```diff
@@ -137,6 +141,26 @@ func (sd SetterDefinition) Filter(object *yaml.RNode) (*yaml.RNode, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	definitions, err := object.Pipe(yaml.Lookup(openapi.SupplementaryOpenAPIFieldName, "definitions"))
```

Comment:
Would this be easier to understand if we moved this part further up in the function, and then let the other pipe calls use the definitions node instead of the root node?

Reply:
Great suggestion. Done. Thanks.